### ISSUE TITLE
Cannot find module '@serverless/fdk'

### ISSUE THREAD

**Issue by scottfoster** [serverless/serverless#4795]
I'm trying to install serverless for the first time, but it gives me an error with serverless/@fdk.

> scott@forest:~$ npm -v
> 5.6.0
> scott@forest:~$ npm install -g serverless
> /usr/local/bin/serverless -> /usr/local/lib/node_modules/serverless/bin/serverless
> /usr/local/bin/sls -> /usr/local/lib/node_modules/serverless/bin/serverless
> /usr/local/bin/slss -> /usr/local/lib/node_modules/serverless/bin/serverless
> 
> > spawn-sync@1.0.15 postinstall /usr/local/lib/node_modules/serverless/node_modules/spawn-sync
> > node postinstall
> 
> 
> > serverless@1.26.1 postinstall /usr/local/lib/node_modules/serverless
> > node ./scripts/postinstall.js
> 
> + serverless@1.26.1
> added 302 packages in 10.031s
> scott@forest:~$ serverless login
> 
>   Serverless Error ---------------------------------------
> 
>   Serverless plugin "/usr/local/lib/node_modules/serverless/lib/plugins/run/index.js" initialization errored: Cannot find module '@serverless/fdk'
> 
>   Get Support --------------------------------------------
>      Docs:          docs.serverless.com
>      Bugs:          github.com/serverless/serverless/issues
>      Forums:        forum.serverless.com
>      Chat:          gitter.im/serverless/serverless
> 
>   Your Environment Information -----------------------------
>      OS:                     darwin
>      Node Version:           9.6.1
>      Serverless Version:     1.26.1

I see that serverless/@fdk is no deprecated, but it's still part of the install script.

**Comment by h3nry0** [comment]
Having the same issue, which is causing the install to error:

    ~$ npm install -g serverless   
    npm http GET https://registry.npmjs.org/serverless
    npm http 304 https://registry.npmjs.org/serverless
    npm WARN engine serverless@1.26.1: wanted: {"node":">=4.0"} (current: {"node":"v0.10.25","npm":"1.3.10"})
    npm http 404 https://registry.npmjs.org/serverless/fdk
    npm ERR! TypeError: Cannot read property 'latest' of undefined
    npm ERR!     at next (/usr/share/npm/lib/cache.js:687:35)
    npm ERR!     at /usr/share/npm/lib/cache.js:675:5
    npm ERR!     at saved (/usr/share/npm/node_modules/npm-registry-client/lib/get.js:142:7)
    npm ERR!     at /usr/lib/nodejs/graceful-fs/polyfills.js:133:7
    npm ERR!     at Object.oncomplete (fs.js:107:15)
    npm ERR! If you need help, you may report this log at:
    npm ERR!     <http://github.com/isaacs/npm/issues>
    npm ERR! or email it to:
    npm ERR!     <npm-@googlegroups.com>
    
    npm ERR! System Linux 3.13.0-55-generic
    npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "serverless"
    npm ERR! cwd /home/user
    npm ERR! node -v v0.10.25
    npm ERR! npm -v 1.3.10
    npm ERR! type non_object_property_load
    npm ERR! 
    npm ERR! Additional logging details can be found in:
    npm ERR!     /home/user/npm-debug.log
    npm ERR! not ok code 0

**Comment by HyperBrain** [comment]
@h3nry0 According to your log:
```
npm WARN engine serverless@1.26.1: wanted: {"node":">=4.0"} (current: {"node":"v0.10.25","npm":"1.3.10"})
```
it seems that your executed node is version `v0.10.25` and npm is version `1.3.10`.

Please make sure that you use a proper node version (useful is Node 6+ although Node 4 also will work).

**Comment by h3nry0** [comment]
Ouch, that's embarrassing - it seems to work fine now that I've updated my mode and npm versions.

For anyone else coming across this, the reason my node and npm versions were so old (0.10.25 for node) is that the apt-get repository is not current. Immediately after provisioning a new Ubuntu 14.04 development environment, I did the usual `apt-get update` and `apt-get upgrade` thinking this would cover me - it doesn't.

Thanks for pointing out my error.
